pyquil/tests/test_quantum_computer.py

```diff
@@ -4,9 +4,10 @@
 import networkx as nx
```

Comment:
This whole test file should eventually be merged with the other `test_quantum_computer.py`, but I wanted the diff to be visible, so we'll do it later